packages/eslint-plugin/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Enahncement
```

Comment:
Out of curiosity, does TypeScript 5 has some breaking changes: https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#breaking-changes-and-deprecations. Do you think it might impact ESLint? Maybe we should follow-up with a breaking change to stay on the safe side? It would also require doing a major version bump for `@wordpress/scripts`.

Reply:
I don't think we need a major update -- I only updated the typescript eslint plugins to the latest minor update (though they do have major updates available). This means the eslint plugins should still support everything they did before, but also have typescript 5 support now. So this shouldn't require people to change anything -- typescript is only a peer dependency of eslint-plugin after all

Reply:
Thank you for the clarification. Works for me 👍🏻